src/components/MySongs/MySongs.js

```diff
@@ -92,7 +96,8 @@ export class MySongsComponent extends SearchFunctionalityProviderComponent {
       onDialogOpen: this.props.onDialogOpen,
       onLegalOptionClick: this.props.onLegalOptionClick,
       playingSongId: this.playingSongId,
-      hasRemoveBtn: true
+      hasRemoveBtn: true,
+      onDragnDrop: this.getNewData.bind(this)
```

Comment:
Call function to fetch the updated data so that next/previous buttons would work after DragNDrop.